src/dmd/initsem.d

```diff
@@ -585,17 +585,23 @@ extern(C++) Initializer initializerSemantic(Initializer init, Scope* sc, ref Typ
 
         auto dil = ci.initializerList[];
 
+        /* If `{ expression }` return the expression initializer
+         */
+        ExpInitializer isBraceExpression()
```

Comment:
Why is this called `isBraceExpression` when it returns an initialiser?

Also why does this not take `dil` an argument? 
`dil.isBraceInitializer()` has a lot more contextual utility.

Reply:
1. Because `isExpInitializer` is already taken. It's braces around an expression.
2. Because it seemed pointless
3. Then I'll change it if it is useful elsewhere. (It's best to expand utility only when there's a specific need for it, otherwise it's just a waste of time. It should be only enough to do its known job. I've wasted a lot of time over-engineering things in the past.)